src/main/java/com/microsoft/spring/data/gremlin/conversion/script/GremlinScriptLiteralGraph.java

```diff
@@ -12,46 +12,44 @@
 import org.springframework.lang.NonNull;
 import org.springframework.lang.Nullable;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
 
 import static com.microsoft.spring.data.gremlin.common.Constants.*;
 
 @NoArgsConstructor
 public class GremlinScriptLiteralGraph implements GremlinScriptLiteral {
 
+    private final GremlinScriptLiteralVertex scriptVertex = new GremlinScriptLiteralVertex();
+
+    private final GremlinScriptLiteralEdge scriptEdge = new GremlinScriptLiteralEdge();
+
     @Override
+    @SuppressWarnings("unchecked")
     public List<String> generateInsertScript(@NonNull GremlinSource source) {
         if (!(source instanceof GremlinSourceGraph)) {
             throw new GremlinUnexpectedSourceTypeException("should be the instance of GremlinSourceGraph");
         }
 
         final List<String> scriptList = new ArrayList<>();
         final GremlinSourceGraph sourceGraph = (GremlinSourceGraph) source;
-        final GremlinScriptLiteralVertex scriptVertex = new GremlinScriptLiteralVertex();
-        final GremlinScriptLiteralEdge scriptEdge = new GremlinScriptLiteralEdge();
+        final List<GremlinSource> vertexes = (List<GremlinSource>) sourceGraph.getVertexSet();
+        final List<GremlinSource> edges = (List<GremlinSource>) sourceGraph.getEdgeSet();
```

Comment:
Why the type cast for List is required?

Reply:
The GremlinSource is generic, if use raw type of GremlinSource, the List<GremlinSource> will be erased to Object.